vxfreeswitch/voice.py

```diff
@@ -118,13 +120,32 @@ def stream_text_as_speech(self, message):
     def get_address(self):
         return self.uniquecallid
 
-    def output_message(self, text):
-        self.log("Sending text as speech: %r" % (text,))
-        return self.stream_text_as_speech(text)
-
-    def output_stream(self, url):
-        self.log("Playing back URL: %r" % (url,))
-        return self.playback(url)
+    def output_message(self, text, settings={}):
+        return self.stream_text_as_speech(text, settings=settings)
+
+    def output_stream(self, message, settings={}):
+        self.log("Playing back: %r" % (message,))
+        if settings.get('barge_in'):
+            terminator = settings.get('wait_for')
+            if settings.get('wait_for') is None:
+                # We just want to get 1 digit
+                minimum, maximum = 1, 1
+                terminator = "' '"
```

Comment:
It makes it as if there is no terminator character. This was back from when when I was using the result from `play_and_get_digits` rather than the existing DTMF things that we have, and having a terminator character meant that if they pressed the terminator character, the result would be an empty string.

This could also be the empty string `"''"`. I'll change it to that and add a comment to make it clearer.